README.md

```diff
@@ -8,17 +8,21 @@ It includes a command-line shell, an associated scripting language and a framewo
 Windows PowerShell is a Windows command-line shell designed especially for system administrators.
 Windows PowerShell includes an interactive prompt and a scripting environment that can be used independently or in combination.
 
-The PowerShell Standard has been created to assist developers create modules and PowerShell hosts which will use only APIs that exist across different versions of PowerShell.
+The PowerShell Standard has been created to assist developers create modules and PowerShell hosts which will run on PowerShell and which will use only APIs that exist across different versions of PowerShell.
```

Comment:
Perhaps we should call out that PowerShell Standard is a reference assembly and does not contain an implementation of PowerShell that can be executed.  This will lead into your statement about the SDK.

Reply:
Addressed